Java/Java Anagrams/Solution2.java

```diff
@@ -0,0 +1,77 @@
+
+//Two strings  and  are called anagrams if they consist same characters, but may be in different orders. So the list of anagrams of  is .
+//
+//Given two strings, print Anagrams if they are anagrams, print Not Anagrams if they are not. The strings may consist at most  English characters; the comparison should NOT be case sensitive.
+//
+//This exercise will verify that you can sort the characters of a string, or compare frequencies of characters.
+//
+//
+//
+//Sample Input 0
+//
+//anagram
+//margana
+
+//Sample Output 0
+//
+//Anagrams
+
+
+
+//Sample Input 1
+//
+//anagramm
+//marganaa
+
+//Sample Output 1:
+//
+//Not Anagrams
+
+
+
+// this is a optimised solution for solving Anagram problems
```

Comment:
this is a optimized solution for solving Anagram problems.
Why is it optimized ?
Ans: The reason is, This solution dosen't look only for the characters and letters in a string for comparison of Anagrams , But it allows our code to look before all the ASCII characters such as letter, numbers , special symbols ets.
And will help finding Anagrams easily with any kind of string problems.